template/web/src/lib/Header.svelte

```diff
@@ -0,0 +1,80 @@
+<script context="module">
+  import { page } from '$app/stores'
+</script>
+<script>
+  export let blogTitle = 'My Blog';
+
+  let path;
+  $:({ path } = $page);
```

Comment:
Whaaaat this is possible?!

Thanks for the trick! I've been wanting to destructure objects in reactive assignments for years - #TIL 🤯

Reply:
Ahhh yes! I do this a lot just to keep templates really readable - super handy and I find it keeps things really declarative esp when a component depends heavily on props